pymeasure/instruments/nf/li5660.py

```diff
@@ -0,0 +1,445 @@
+        """,
+        validator=strict_discrete_set,
+        values={"High": "HIGH", "Medium": "MED", "Low": "LOW"},
+        map_values=True
+    )
+
+    filter_slope = Instrument.control(
+        ":FILT:SLOP?", ":FILT:SLOP %d",
+        """ A property that control the filter attenuation slope (primary PSD). """,
+        validator=strict_discrete_set,
+        values=[6, 12, 18,24],
+        map_values=False
+    )
+
+    time_constant = Instrument.control(
+        ":FILT:TCON?",":FILT:TCON %g",
+        """ A property that control the filter time constant (primary PSD). """,
+        validator=truncated_range,
+        values=[1e-6, 50e+3]
+    )
+
+    filter_type = Instrument.control(
+        ":FILT:TYPE?",":FILT:TYPE %s",
+        """ A property that control the filter type (primary PSD). """,
+        validator=strict_discrete_set,
+        values={"Time Constant": "EXP", "Synchronous": "MOV"},
+        map_values=True
+    )
+
+    smoothing = Instrument.control(
+        ":NOIS?",":NOIS %d",
+        """ A property that control the output smoothing coefficient for noise density measurement. """,
+        validator=strict_discrete_set,
+        values=[1, 4, 16, 64]
+    )
+
+    ####################
+    # Calculate / Data #
+    ####################
+    display_format = Instrument.control(
+        ":CALC1:FORM?",":CALC1:FORM %s",
+        """ A property that allow user to set the measurement parameters to be displayed and output as DATA1. """,
+        validator=strict_discrete_set,
+        values={"Real": "REAL",
+                "Mlinear": "MLIN",
+                "Imaginary": "IMAG",
+                "Phase": "PHAS",
+                "Noise": "NOIS",
+                "Aux1": "AUX1",
+                "Real2": "REAL2",
+                "Mlinear2": "MLIN2"},
+        map_values=True
+    )
+
+    reference_current = Instrument.control(
+        ":CALC1:MATH:CURR?",":CALC1:MATH:CURR %g",
+        """ A property that allow user to set the current reference value for normalize calculation. """,
+        validator=truncated_range,
+        values=[1e-15, 1e-6]
+    )
+
+    calculation_format = Instrument.control(
+        ":CALC1:MATH:EXPR:NAME?", ":CALC1:MATH:EXPR:NAME %s", 
+        """ A property that allow user to normalize calculation format. """,
+        validator=strict_discrete_set,
+        values={"DB": "DB, 'dB'",
+                "Reference": "PCNT, ''",
+                "Sensitivity": "PCFS, ''",
+                },
+        map_values=True
+    )
+
+    reference_voltage = Instrument.control(
+        ":CALC1:MATH:VOLT?", ":CALC1:MATH:VOLT %g",
+        """ A property that allow user to set the voltage reference value for normalize calculation. """,
+        validator=truncated_range,
+        values=[1e-9, 1e+1]
+    )
+
+    offset_enable = Instrument.control(
+        ":CALC1:OFFS:STAT?", ":CALC1:OFFS:STAT %s",
+        """ A property that allow user to enable/disable offset for the primary PSD's X output. """,
```

Comment:
It would be nice to (explicitely) mention the possible values in the docstring. You could also consider using booleans rather than the "On" and "Off" strings.